Main.java

```diff
@@ -0,0 +1,378 @@
+/* Machine Coding of SplitWise by TUSHAR ANAND, KIIT DEEMED TO BE UNIVERISTY*/
+
+
+/* Importing Necessary Packages*/
+
+import java.util.Random;
+import java.util.*;
+import java.io.*;
+import java.lang.*;
+public class Main
+{
+	/** 
+	In this project we are maintaining a two dimensional matrix to keep the record of transaction between the users
+	The matrix is named as 'Matrix' and we will call it as Transaction Matrix.
+	**/
+
+	
+	/*
+	printTable() method print the Matrix of Transction done between the users.
+	It keep records from the begining meaning if there is no borrow between the users still we maintain the amount.
+	**/
+	
+	public static void printTable(int Matrix[][])
+	{
+		System.out.println();
+		for(int i=1;i<Matrix.length;i++)
+		{
+			for(int j=1;j<Matrix[i].length;j++)
+			{
+				System.out.print((Matrix[i][j]+0.00)+" ");
+			}
+			System.out.println();
+
+		}
+	}
+
+
+	/**
+	updateTransctionTableEqual() method updates the Transaction Matrix on 'EQUAL' option 
+	**/
+
+	public static void updateTransctionTableEqual(int Matrix[][],String  payer, String whom[] , int amount)
+	{
+		int split_wise = (amount)/whom.length;
```

Comment:
Either use camelCase or underscore_separated based on the language’s convention. Please do not use both in your solution.
Source: [How to prepare for machine coding round?](https://workat.tech/machine-coding/article/how-to-prepare-for-machine-coding-round-naf2ih7a9e5l)